### PR TITLE
Missing DTL Import DLL in DTA Task for Register-Environment

### PR DESCRIPTION
**Pull request by nigurr**
Issue: Missing DTL Import DLL in DTA Task for Register-Environment cmdlet.
Fix: Importing the DTL dll explicitly in DTA Task

### CODE REVIEW

**Comment by nigurr** [comment]
Internal tracking details: 531337

**Comment by nigurr** [comment]
Manual testing done and verified the fix.

**Comment by kaadhina** [comment]
Looks good.

File: Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1 [hunk 1]
```diff
@@ -68,6 +68,7 @@ Write-Verbose "VerifyTestMachinesAreInUseScriptLocation = $verifyTestMachinesAre
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.DTA"
+import-module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
```
Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: looks fine. we need to see why is it working on new agent scenarios.

Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: also see if how we can catch this before.

Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1]: also remember to bump up the nugget version so that other sus get it by default

**Pull request merged**